config/channel/resources/kafkachannel-crd.yaml

```diff
@@ -39,15 +39,18 @@ spec:
               type: object
               properties:
                 numPartitions:
-                  description: NumPartitions is the number of partitions of a Kafka topic. By default, it is set to 1.
+                  description: 'NumPartitions is the number of partitions of a Kafka topic. By default, it is set to 1.'
                   type: integer
                   format: int32
                   default: 1
                 replicationFactor:
-                  description: ReplicationFactor is the replication factor of a Kafka topic. By default, it is set to 1.
+                  description: 'ReplicationFactor is the replication factor of a Kafka topic. By default, it is set to 1.'
```

Comment:
why do we add the `'` only to those two? Should other description do that too - for openAPI? If so. let's do that separately

Reply:
Yeah, you're right... I noticed that some of the `description` fields were quoted and didn't bother to understand why you quote things in YAML.  Seems it's only recommended when necessary to prevent values from being misinterpreted as fields (when they contain a colon or similar characters).  There are probably some `description` fields that could be unquoted but, as you said we can leave that for another time and keep these changes to the bare minimum.  thanks!